qiskit/result/distributions/quasi.py

```diff
@@ -38,8 +38,7 @@ def __init__(self, data, shots=None):
                 The keys can be one of several formats:
 
                     * A hexadecimal string of the form ``"0x4a"``
-                    * A bit string prefixed with ``0b`` for example
-                      ``'0b1011'``
+                    * A bit string e.g. ``'0b1011'`` or ``"01011"``
```

Comment:
Do you want to update the docstring on Probabilities too? It behaves the same way.